modules/install-acs-operator-cloud.adoc

```diff
@@ -10,7 +10,7 @@ Using the OperatorHub provided with {ocp} is the easiest way to install the {pro
 
 .Prerequisites
 * You have access to an {ocp} cluster using an account with Operator installation permissions.
-* You must be using {ocp} 4.10 or later.
+* You must be using {ocp} 4.10 or later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```

Comment:
We will be required to bump `4.10` again when the version support matrix changes. I suggest introducing a variable like `{minimum-ocp-version}` or better phrasing it in a way we don't need to revisit the version number at all.
Something like
```
You must be using {ocp} of supported version. See link for ACS version support policy.
```

Reply:
Confirming resolved. There are more places to use `{rhacs-operator-min-ocp-version}` though. Highlighted them with comments.